todo/models.py

```diff
@@ -133,7 +133,8 @@ class Comment(models.Model):
     """
 
     author = models.ForeignKey(
-        settings.AUTH_USER_MODEL, on_delete=models.CASCADE, blank=True, null=True
+        settings.AUTH_USER_MODEL, on_delete=models.CASCADE, blank=True, null=True,
+        related_name="todo_comments"
```

Comment:
I would think this change would break todo code in at least one place, no? Will test soon. But the system still works for you?

Reply:
All existing tests passed.  The code is running live an in use.  And there are no references to comment_set, the default related_name anywhere in the code base.  Looks safe to me.

Reply:
Excellent, thank you for looking.